.cirrus.yml

```diff
@@ -0,0 +1,60 @@
+env:
```

Comment:
I guess this is here to do builds on the systems that github doesn't currently support?

Whatever the reason is, using two CI systems is worth documenting, probably in comments in both files. Otherwise a future developer might make changes to one CI script but not the other.

Reply:
correcto, it is currently only open source and free to use m1 runners, they also provide arm64 linux runners (as well as freebsd), and coupled with the ability to run the workflows locally in ephemeral VM's, it makes for a pleasant experience

Circle CI offers arm64 linux, but only x86_64 macs, so cirrus is simple free way we can get arm64 runners for linux and macos, and also allow contributors to perform the same runs, on forks, with cirrus too.

If we got AWS runners or something like that, it would exclude users who don't have the ability to pay for that kind of compute.

Reply:
Will add some developer docs to cover this platform in an upcoming pr :)